tool/README.md

```diff
@@ -93,20 +83,46 @@ git pull upstream master
 
 #### Publish the packages
```

Comment:
can you add a note that in order to publish, you need to temporarily comment out dependency_overrides?

Reply:
no because my next cleanup CL is to have the publish script do that for us.